models/oauth.schema.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require("mongoose");
+
+// Schema
+const postOauthSchema = new mongoose.Schema(
+  {
+    token: {
```

Comment:
We don't need save OauthToken. Token it self will contain user id. So we don't need this model.